sky/shell/ui/engine.h

```diff
@@ -98,6 +104,9 @@ class Engine : public UIDelegate,
   std::string country_code_;
   mojo::Binding<SkyEngine> binding_;
 
+  // Asset bundle backed by the FLX zip file (will be deleted by Mojo)
+  mojo::asset_bundle::ZipAssetBundle* zip_asset_bundle_;
```

Comment:
Do we need to deal with locale-dependent font fallback chains?  (CJK,
perhaps emoji)

On Wed, Feb 17, 2016, 10:32 AM Adam Barth <notifications@github.com> wrote:

> In sky/shell/ui/engine.h
> <https://github.com/flutter/engine/pull/2402#discussion_r53208652>:
>
> > @@ -98,6 +104,9 @@ class Engine : public UIDelegate,
> >    std::string country_code_;
> >    mojo::Binding<SkyEngine> binding_;
> >
> > +  // Asset bundle backed by the FLX zip file (will be deleted by Mojo)
> > +  mojo::asset_bundle::ZipAssetBundle* zip_asset_bundle_;
>
> The lifetime of this field is pretty sketchy. How do we know that the
> object is alive for the period of time that we use it? Maybe we should
> create the font selector eagerly (i.e., before DidCreateIsolate) and only
> install it into the isolate when DidCreateIsolate happens?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/engine/pull/2402/files#r53208652>.
>